src/common.js

```diff
@@ -397,3 +410,17 @@ function _register(name, implementation, wrapOptions) {
   }
 }
 exports.register = _register;
+
+exports.getProcessEnv = function () {
+  var env = objectAssign({}, process.env);
+
+  if (config.nodepath) {
+    env[PATH] += (process.platform === 'win32' ? ';' : ':') + path.join(process.cwd(), 'node_modules', '.bin') + (process.platform === 'win32' ? ';' : ':');
+  }
+
+  return env;
+};
+
+exports.getPathFromEnv = function (env) {
```

Comment:
At some places in the code base we have used `process.env.path || process.env.Path || process.env.PATH;` to get the correct path. I just made a function to keep it at one spot.